desktop/components/react/responsive_window/test/responsive_window_spec.js

```diff
@@ -0,0 +1,37 @@
+import 'jsdom-global/register'
```

Comment:
This is simpler and more modern than `benv`, and can use seamlessly with http://airbnb.io/enzyme/docs/api/mount.html to simulate a slender virtual browser env in node.